app.py

```diff
@@ -38,34 +44,46 @@
     }
 
     config = None
-
-    with open('config.yml', 'r') as config_file:
-        try:
-            config = yaml.load(config_file)
-        except yaml.YAMLError as err:
-            print(err)
-            exit(1)
-
+    config_path = None
+    if args.config is not None:
+        config_path = args.config
+    else:
+        config_path = 'config.yml'
+
+    try: 
+        with open(config_path, 'r') as config_file:
+            try:
+                config = yaml.load(config_file)
+            except yaml.YAMLError as err:
+                print(err,file=sys.stderr)
+                exit(1)
+    except FileNotFoundError as err:
+        print('No such config file', file=sys.stderr)
+        exit(1)
+    
     if config is None:
-        print('Error with your config')
+        print('Error with your config', file=sys.stderr)
         exit(1)
-    elif 'pb_token' not in config:
-        print('Config must contain your Pushbullet Access Token!')
+    elif use_pushbullet == True and 'pb_token' not in config:
```

Comment:
This can just be `elif use_pushbullet and 'pb_token' not in config` since use_pushbullet is already a boolean. There are also a few other places below with the same thing when checking for pushbullet and libnotify.

Reply:
ack